Src/Plugins/Tests.Security/Validators/SEC101_050.IdentifiableNpmAuthorTokenValidatorTests.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Helpers;
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+using Newtonsoft.Json;
+
+using Xunit;
+
+using static Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.IdentifiableNpmAuthorTokenValidator;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
+{
+    public class IdentifiableNpmAuthorTokenValidatorTests
```

Comment:
>IdentifiableNpmAuthorTokenValidatorTests [](http://example.com/codeflow?start=17&length=40)

once u create the helper, u might be able to merge the test logic for both rules.